tests/helpers.py

```diff
@@ -56,3 +56,15 @@ def config_all_links_up(self):
     def stop(self):
         self.net.stop()
         #mininet.clean.cleanup()
+
+    def change_node_status(self, status, target='tcp:127.0.0.1:6654'):
+        node = self.net.get('Ampath1')
```

Comment:
Gretel, thanks for changing the function to be on the helpers! this will be very nice and it will allow reuse if others are needed.

Let me ask you one more thing please: right now the node which will have the connection changed is hard coded above. :( can you please make this into a parameter? Also, it seems like the "status" parameter is not quite useful. I mean: as long as you call the function with a valid target, this will become "status UP". So maybe we can remove the status parameter.